lib/pure/asyncdispatch.nim

```diff
@@ -9,11 +9,12 @@
 
 include "system/inclrtl"
 
-import os, tables, strutils, times, heapqueue, options
-
+import os, tables, strutils, times, heapqueue, options, asyncstreams
+import asyncfutures except callSoon
 import nativesockets, net, deques
 
 export Port, SocketFlag
+export asyncfutures, asyncstreams
```

Comment:
Yes, it is exported (callSoon is exported from asyncfutures, so `export asyncfutures` re-exports it).